system/include/emscripten/html5.h

```diff
@@ -64,17 +64,16 @@ extern "C" {
 #define EMSCRIPTEN_EVENT_BEFOREUNLOAD          28
 #define EMSCRIPTEN_EVENT_BATTERYCHARGINGCHANGE 29
 #define EMSCRIPTEN_EVENT_BATTERYLEVELCHANGE    30
-#define EMSCRIPTEN_EVENT_WEBGLCONTEXTLOST      31
-#define EMSCRIPTEN_EVENT_WEBGLCONTEXTRESTORED  32
 #define EMSCRIPTEN_EVENT_MOUSEENTER            33
 #define EMSCRIPTEN_EVENT_MOUSELEAVE            34
 #define EMSCRIPTEN_EVENT_MOUSEOVER             35
 #define EMSCRIPTEN_EVENT_MOUSEOUT              36
 #define EMSCRIPTEN_EVENT_CANVASRESIZED         37
 #define EMSCRIPTEN_EVENT_POINTERLOCKERROR      38
 
+#ifndef EMSCRIPTEN_RESULT
```

Comment:
instead of these ifndefs, how about making `html5_webgl.h` include `html5.h`? seems safer and simpler.

Reply:
Wanted to avoid introducing cross-includes, but I suppose that is simpler - changed to that instead.